kubernetes/linux/setup.sh

```diff
@@ -57,10 +57,10 @@ sudo apt-get install jq=1.5+dfsg-2 -y
 #used to setcaps for ruby process to read /proc/env
 sudo apt-get install libcap2-bin -y
 
-wget https://dl.influxdata.com/telegraf/releases/telegraf-1.24.2_linux_$ARCH.tar.gz
-tar -zxvf telegraf-1.24.2_linux_$ARCH.tar.gz
+wget https://dl.influxdata.com/telegraf/releases/telegraf-1.25.1_linux_$ARCH.tar.gz
```

Comment:
Can we move this to a variable? So, we dont have to change in multiple places

Reply:
changed